RUNNING_DOCKER.RST

```diff
@@ -0,0 +1,369 @@
+
+::
+    
+    docker -v
+
+You should see something like the following:
+
+``Docker version 19.03.4, build 9013bf5``
+
+To download and install docker-compose please visit: https://docs.docker.com/v17.09/compose/install/
+and follow the installation instructions to download docker-compose if needed.
+To verify you have successfully installed docker-compose type:
+
+::
+
+    docker-compose -v
+
+You should see something like the following):
+
+``docker-compose version 1.16.1, build 6d1ac219``
+
+Running MetaCI In Docker:
+=========================
+
+To run MetaCI on Docker there are 3 major steps necessary:
+----------------------------------------------------------
+
+- `.env File Creation and Variable Declaration`_
+.. _.env File Creation and Variable Declaration: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#env-file-creation-and-variable-declaration
+
+- `Building Your Docker Containers`_
+.. _Building Your Docker Containers: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#building-your-docker-containers
+
+- `Running Your Docker Containers`_
+.. _Running Your Docker Containers: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#running-your-docker-containers
+
+
+.env File Creation and Variable Declaration
+-------------------------------------------
+
+*Please begin by making a copy of env.example and renaming it .env in your root project directory*
+
+Variables Required Locally
+---------------------------------
+
+DJANGO_SECRET_KEY: 
+    This represents the secret key for the django web application, for local testing, 
+    arbritary strings such as the one given in the env.example will suffice. 
+    Otherwise use your production secret key.
+
+POSTGRES_USER: 
+    Environment variable set in the docker-compose.yml file under the postgres service, 
+    represents database user. This value has already been configured for you unless 
+    you decide to reconfigure it.
+
+POSTGRES_PASSWORD: 
+    Environment variable set in the docker-compose.yml file under the postgres service,
+    represents database password.This database is configured with no password for 
+    development purposes so leave as is unless changing for production purposes.
+
+POSTGRES_DB:
+    Environment variable set in the docker-compose.yml file under the postgres service,
+    represents database. This variable has already been set to the proper 
+    value `metaci` for the user.
+
+GITHUB_USERNAME:     
+    This represents the username of either the tester or service account configured for MetaCI
+
+GITHUB_PASSWORD:      
+    This represents the password or personal access token a user must have to access 
+    their account a personal access token will be used when Multi Factor Authentication is enabled.
+
+If you need to generate a personal access token please visit the following: 
+https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line
```

Comment:
I would preface the github settings with "MetaCI needs a connection to the GitHub API to fetch repositories and create releases. This can be set up using a personal GitHub account by setting GITHUB_USERNAME and GITHUB_PASSWORD, or using a GitHub App by setting GITHUB_APP_ID and GITHUB_APP_KEY." (Or we can leave that 2nd line out; it should be obvious in env.example)